crawl-ref/source/dat/des/arrival/large.des

```diff
@@ -1115,3 +1115,57 @@ MAP
       xxxxxxGxxxxxx
         xxxxxxxxx
 ENDMAP
+
+##############################################################################
+# 39x22
+NAME:   beargit_arrival_diablo_church
+TAGS:   arrival no_trap_gen no_item_gen no_monster_gen
+ORIENT: float
+MONS:   rat / nothing w:50
+FTILE:  `t = floor_grass, 1'G_> = floor_limestone
+KPROP:  ` = no_tele_into
+KFEAT:  _ = altar_ecumenical
```

Comment:
This doesn't look like it places anywhere?

Unless its placing behind the glass, I would remove this line; while ecumenical altars do spawn on D:1 quite regularly after the recent change, putting them guaranteed in a vault makes start scamming too easy.

Reply:
yeah, that was missed when I removed the imps